[156] Style the Contact Us Page using CSS

OluwaboriShobayo opened the pull request:
![image](https://user-images.githubusercontent.com/74042089/101979607-b5d29000-3c5e-11eb-8a18-d31645a740bf.png)
![image](https://user-images.githubusercontent.com/74042089/101979620-c08d2500-3c5e-11eb-8ed5-677f37bc80ed.png)

![image](https://user-images.githubusercontent.com/74042089/101979626-d7cc1280-3c5e-11eb-8c0a-a73e448aca0f.png)
![image](https://user-images.githubusercontent.com/74042089/101979631-e31f3e00-3c5e-11eb-92b4-4d1f20a0a7cf.png)

![image](https://user-images.githubusercontent.com/74042089/101979642-f3371d80-3c5e-11eb-8c1b-ba481b2860b0.png)

adaugochi commented on submissions/oluwaboriShobayo/CSS/contactUs.css:
```diff
@@ -0,0 +1,228 @@
+     #header {
+            display:flex;
+            justify-content:space-between;
+            /* padding:20px;
```
I think you forgot to remove this comments

adaugochi commented on submissions/oluwaboriShobayo/CSS/productPage.css:
```diff
@@ -0,0 +1,237 @@
+/* This section is for the media screen query*/
```
This task is for contact us page. Why are we have styling for product page? Kindly remove it

adaugochi commented on submissions/oluwaboriShobayo/CSS/signUp.css:
```diff
@@ -0,0 +1,289 @@
+
```
This task is for contact us page. Why are we have styling for signup page? Kindly remove it

adaugochi commented on submissions/oluwaboriShobayo/HTML/Laptop media query.css:
```diff
@@ -0,0 +1,197 @@
+@media  only screen and (min-device-width: 1200px) {
```
What is this CSS style for? Meanwhile, all CSS file should be inside your CSS folder

adaugochi commented on submissions/oluwaboriShobayo/HTML/Laptopmedia-query.css:
```diff
@@ -0,0 +1,199 @@
+@media  only screen and (min-device-width: 1200px) {
```
Same here too

adaugochi commented on submissions/oluwaboriShobayo/HTML/Mediaquer-Tab.css:
```diff
@@ -0,0 +1,79 @@
+ @media  only screen and (max-device-width: 768px) {
```
And this too

adaugochi reviewed (changes requested):
All images should be placed inside the `images` folder that is in your HTML folder

Also, remove spaces on file/folder names.

From what I am seeing it as if you did 3 tasks all at once. You are meant to do one task at a time

adaugochi reviewed (changes requested):
These images are still in `html-forms` folder. Kindly move them to their appropriate folders 
![image](https://user-images.githubusercontent.com/43061141/102075389-75157b00-3e06-11eb-9faa-100143e3c51e.png)
Also. the images you use here is too much. are you sure you are using all of them?

Just so you know, you deleted your productPage.html. Is it intentional?
![image](https://user-images.githubusercontent.com/43061141/102075531-a8f0a080-3e06-11eb-918b-6d4a483ade17.png)

OluwaboriShobayo commented:
Thanks Ada. I moved the productPage and the signUp page to another location because I felt that it might be causing confusion. I plan to add it back after the contactUs page has been merged.

Also, I have deleted the images I am not using.
